README.adoc

```diff
@@ -0,0 +1,177 @@
+= Eclipse uProtocol client vsomeip python
+:toc:
+
+Python uPClient implementation for SOME/IP using vsomeip
+
+== Overview
+
+Python module to leverage someip implementations developed from other programming languages.
+
+
+== Getting Started
+
+=== Prerequisites
+Download, Build, and Install vsomeip - [COVESA / vsomeip](https://github.com/COVESA/vsomeip)
+for windows support: [COVESA / vsomeip (fork)](https://github.com/justinlhudson/vsomeip)
```

Comment:
Update this url

Reply:
This is related to a Windows OS patch made by Justin for main C++ vsomeip Repo. He has made a PR to COVESA vsomeip repo. Once its merged, this can be updated, until then this remains